cli/js/compiler_util.ts

```diff
@@ -90,18 +90,15 @@ function cache(
   }
 }
 
-let OP_FETCH_ASSET: number;
 const encoder = new TextEncoder();
 const decoder = new TextDecoder();
 
 /** Retrieve an asset from Rust. */
 export function getAsset(name: string): string {
-  if (!OP_FETCH_ASSET) {
-    OP_FETCH_ASSET = core.ops()["fetch_asset"];
-  }
+  const opId = core.ops()["fetch_asset"];
```

Comment:
slightly inefficient in that `core.ops()` will get called on every asset fetch during bundling...

Reply:
Yeah.... There's a problem where if you try to cache the op id it will be done during snapshot and then have a different value during runtime. I guess this isn't measurable so I'd like it as is... but it would be nice to avoid.